CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json

```diff
@@ -0,0 +1,17 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Jiazhuo Wang, Haofeng Kou, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, jiazhuowang@baidu.com, haofengkou@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch torch-1.0.1.post2-cp35-cp35m-linux_x86_64.whl",
+	    "codeURL": " https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8, commitHash: https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8/commit/8678e0bcc72e6b72d5eb66e997e1ae92f2eaf2a1",

```

Comment:
Please split `commitHash` into a separate field or use the Github URL with the commit hash in it.

Reply:
OK, please refer to the updated version, thanks.